include/kernels/dslash_staggered.cuh

```diff
@@ -6,17 +6,19 @@
 #include <color_spinor.h>
 #include <dslash_helper.cuh>
 #include <index_helper.cuh>
+#include <kernels/laplace.cuh>
```

Comment:
Not quite obvious why this is included here

Reply:
Should have been dslash_pack.cuh, fixed.